### DISCUSSION TITLE
Playback Reporting Plugin

### DISCUSSION THREAD

**Discussion by DuvelCorp** [CyferShepard/Jellystat#196] [Q&A] [answered]
Hi

I just installed Jellystat and it is great
I have imported my Playback Reporting Plugin Data with the settings button and everything seems to be there.

I guess this is only a oneshot operation ?

Also I'd like to know if there's a purpose to let the pluggin running on Jellyfin if I definitively adopt Jellystats? Ie does it logs things that Jellysstas doesnt ?

Cheers

**Comment by CyferShepard** [comment, marked as answer]
Hey @DuvelCorp 

Yea its pretty much meant to be a oneshot operation. Once the data is imported, Jellystat does its own tracking with alot more data granularity. This means you can safely stop using the Playback plugin if you want. Jellystat was built where tracking is basically 1 to 1 in regards to the Plugin, but with improvements over just using the plugin.

**Comment by vw-kombi** [comment]
I just saw this task - and was wondering if this was possible.
I am currently using jellystat with emby however.
I use playback reporting plugin in emby.
It si the same for me to just click this button ?
It runs in about 2 seconds - and I dont get my previous 3 years data.
Does it not working with the emby option - or do I have to do something extra ?
Thanks.

**Comment by CyferShepard** [comment]
Hey @vw-kombi , emby support is still being worked on, some functionality regarding that does not work yet. You can track progress on this in issue #133
